tests/apps/courses/test_cms_plugins_program.py

```diff
@@ -204,20 +204,18 @@ def test_cms_plugins_program_fallback_when_never_published(self):
         # Program's name should be present as a link to the cms page
         self.assertContains(
             response,
-            '<a href="/en/programme-publique/" class="program-glimpse program-glimpse--link" >',
+            '<a href="/en/programme-publique/" class="program-glimpse__link"',
```

Comment:
This test doesn't pass locally and I can't figure out why.

When I debug the response, I get:

```
<a href="/en/programme-publique/"
									class="program-glimpse__link program-glimpse__stretched-link"> programme publique
								</a>
```

So, I see why this would break because there are tab characters between the end of the `href` and the `class`. But seeing my template, I don't get why I have those tabs... Is there maybe some test caching I could have that breaks stuff?

Reply:
Strange, from my side all is working fine 🤔

Reply:
same as above, use lxml+cssselect for this test?